### DISCUSSION TITLE
Github Sponsors able in Thailand?

### DISCUSSION THREAD

**Discussion by mrthiti** [community/community#27414] [Other Feature Feedback, Questions, & Ideas]
I would to get start with Github Sponsors, but, I have problem about register Stripe that it not have Thailand in country list.

**Comment by nestiank** [comment]
South Korea also...
I think GitHub Sponsors in countries where Stripe is unavailable are practically unavailable.

Stripe in Thailand is now on beta.
You can request beta invitation in this page: https://stripe.com/global#TH

**Comment by Helloyunho** [comment]
Same in Korea
I got the GitHub Sponsor invitation today and I was so excited until I see that Stripe is the only option available for payment 😭
At least as above said Thailand is in beta so you will be able to sign up someday soon but no hopes for Korea for now 😢

**Comment by jfullstackdev** [comment]
yup, Philippines too, I received the message this morning but maybe they are still working on it 

as you can see here, we are not yet included, let's wait and see 

![image](https://user-images.githubusercontent.com/47092464/181506833-35e12086-62ea-4abc-9d40-f6948650fa19.png)

**Comment by jfullstackdev** [comment]
update by now, Philippines is already okay

check this out

https://github.com/orgs/community/discussions/8712

try it again for Thailand and South Korea

**Comment by Helloyunho** [comment]
Korea also works now! Thanks for checking @jdevstatic 😘

**Comment by jfullstackdev** [comment]
please take note that it's not a separate Stripe account, it will be created through GitHub Sponsors,

**Comment by jfullstackdev** [comment]
someone needs help here about the Stripe account, if you know the answer, please tell us thanks 

https://github.com/orgs/community/discussions/27546

**Comment by mrthiti** [comment]
From now,Stipe available for Thailand.

Thank you.
